x-pack/legacy/plugins/uptime/public/hooks/update_kuery_string.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { combineFiltersAndUserSearch, stringifyKueries } from '../lib/helper';
+import { esKuery } from '../../../../../../src/plugins/data/common/es_query';
+import { store } from '../state';
+import { setEsKueryString } from '../state/actions';
+
+export const useUpdateKueryString = (indexPattern: any, search: string, urlFilters: string) => {
```

Comment:
Might be nice to explicitly declare the expected return values of this function, just for readability / maintainability. It should be `[string, Error?]` I believe

Reply:
agreed, done.